state/execution_test.go

```diff
@@ -218,7 +218,11 @@ func TestUpdateValidators(t *testing.T) {
 
 	for _, tc := range testCases {
 		t.Run(tc.name, func(t *testing.T) {
-			_, err := updateValidators(tc.currentSet, tc.abciUpdates)
+			updates, err := types.PB2TM.ValidatorUpdates(tc.abciUpdates)
+			if err != nil {
```

Comment:
```suggestion
assert.NoError(t, err)
```

`if err != nil` can be dropped.